infra/__main__.py

```
@@ -63,11 +63,10 @@
 
 # Import other necessary components
 try:
-    from infra.components import lambda_layer  # side effects
+    from infra.components import lambda_layer  # noqa: F401 (side effects)
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)